plenum/common/messages/fields.py

```diff
@@ -263,8 +254,8 @@ def _specific_validation(self, val):
 # TODO: think about making it a subclass of Base58Field
 class VerkeyField(FieldBase):
     _base_types = (str, )
-    _b58short = Base58Field(short=True)
-    _b58long = Base58Field(long=True)
+    _b58short = Base58Field(decodedLengthConstraints=(16,))
+    _b58long = Base58Field(decodedLengthConstraints=(32,) )
 
     def _specific_validation(self, val):
         vk_error = NonEmptyStringField().validate(val)
```

Comment:
Looks like it is an extra check because an empty string has byte length 0 which we're gonna check below.